src/Layout.soy

```diff
@@ -4,7 +4,7 @@
 * @param data
 */
 {template .render}
-	<div class="layout">
+	<div class="layout container-fluid">
```

Comment:
People may wish to use `container` instead of `container-fluid`, and it can be added on a common parent of layout and other components, so I think it's best not to add it here, allowing this to be more flexible. Do you think that makes sense?